src/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -157,6 +160,11 @@
      */
     private int alertCount;
 
+    /**
+     * New alerts (with new description) found during the scan.

```

Comment:
`with new description` is misleading, we check more than that. Maybe say `in the current session`?